devops/internal_servers/servers_and_services/provided_services/index.rst

```diff
@@ -10,14 +10,14 @@ Provided Services
 The following table shows what server runs which service. Some of the services are passive.
 That means you do not use them active, e.g. The Teamcity-Agents.
 
-   ================================= ================== ================== ==================
-    host03a                           host03b            host03c            host03d
-   ================================= ================== ================== ==================
-    Maven (100)                       JAVA-WS (100)      Postgres (111)     TC-Agent-6 (102)
-    File server (101)                 Edge (101)         TC-Agent-4 (116)   TC-Agent-7 (2007)
-    Nagios (107)                      Kibana (118)                          TC-Agent-8 (2008)
+   =================================== =================================== =================================== ===================================
+    `host03a <https://host03a:8006/>`_  `host03b <https://host03b:8006/>`_  `host03c <https://host03c:8006/>`_  `host03d <https://host03d:8006/>`_
```

Comment:
https://host03a:8006/ →https://host03a.tocco.ch and same change for the other hosts. For that we have a valid SSL cert.